src/navigation/HowItWorksStack.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent } from "react"
+import { createStackNavigator } from "@react-navigation/stack"
+
+import { Stack as DestinationStack } from "./index"
+import HowItWorksScreen from "../HowItWorks/HowItWorksScreen"
+import useHowItWorksData, {
+  HowItWorksScreenDatum,
+} from "../HowItWorks/useHowItWorksData"
+
+const Stack = createStackNavigator()
+
+interface HowItWorksStackProps {
+  destinationOnSkip: DestinationStack
+}
+
+const HowItWorksStack: FunctionComponent<HowItWorksStackProps> = ({
+  destinationOnSkip,
+}) => {
+  const howItWorksData = useHowItWorksData(destinationOnSkip)
```

Comment:
Why are we using a hook here? is it mostly to group the data and get access to the use useTranslationHook?

Typically hooks are intended for sharing stateful logic, which we arent really needing to do here beyond wrapping the useTranslation hook. So it seems like we are adding a bit of complexity for no real gain.  What do you think about inlining the screen data?

Reply:
Makes sense! 👍